modules/nodes-pods-configuring-replication.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-configuring.adoc
+
+[id='nodes-pods-configuring-replication_{context}']
+= Controlling pod counts using replication controllers
```

Comment:
This sounds like the title of a procedure, not a concept. (I think it's a good concept that needs a new title!)